ui/components/Notification/MesheryEventViewer.js

```diff
@@ -0,0 +1,20 @@
+import MesherySnackbarWrapper from "./MesherySnackbarWrapper"
+
+
+
+function MesheryEventViewer () {
```

Comment:
let's get into the habit of adding a short description as to what this component is supposed to do just above it's definition